example/Example.res

```diff
@@ -57,7 +57,9 @@ module Form = {
         render={({field: {name, onBlur, onChange, ref, value}}) =>
           <div>
             <label> {name->React.string} </label>
-            <input name onBlur onChange ref value />
+            <input
+              name onBlur={_event => onBlur()} onChange={Controller.handleEvent(onChange)} ref value
```

Comment:
The Example is not relevant as in a "real world" project most of the native inputs would be nested in a few components. So the `onBlur={_event => onBlur()} onChange={Controller.handleEvent(onChange)}` boilerplate shouldn't be that annoying and is worth it considering the benefits.